doc/nrf/config_and_build/hwmv2_migration.rst

```diff
@@ -0,0 +1,57 @@
+.. _hwmv1_to_v2_migration:
```

Comment:
This document should go in the migration folder

Reply:
I agree.